<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuConfigWidget</class>
 <widget class="QWidget" name="NeuConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">神经元配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="NeuClass_LB">
       <property name="text">
        <string>神经元类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="NeuClass_CB">
       <item>
        <property name="text">
         <string>HH</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LIF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Izhikevich</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreateCmd_BT">
       <property name="text">
        <string>生成命令</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="10">
    <widget class="NeuPaint" name="NeuWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>330</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="NeuCount_LB">
       <property name="text">
        <string>神经元数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="NeuCount_SL">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="singleStep">
        <number>0</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="NeuCount_SB">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="SynWeight_LB">
       <property name="text">
        <string>突触权重</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="SynWeight_SL">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="SynWeight_SB">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="SynRatio_LB">
       <property name="text">
        <string>突触连接比例</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="SynRatio_SL">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="SynRatio_SB">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="CpuCount_LB">
       <property name="text">
        <string>处理器数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="CpuCount_SL">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="CpuCount_SB">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="R_LB">
       <property name="text">
        <string>电阻</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="R_SL">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="R_SB"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="C_LB">
       <property name="text">
        <string>电容</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="C_SL">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="C_SB"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="C_LB_2">
       <property name="text">
        <string>注入电流</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="C_SL_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="C_SB_2"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="C_LB_3">
       <property name="text">
        <string>仿真步长</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="C_SL_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="C_SB_3"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="C_LB_4">
       <property name="text">
        <string>阈值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="C_SL_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="C_SB_4"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NeuPaint</class>
   <extends>QWidget</extends>
   <header location="global">neupaint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
